docs/components/VaContent.md

```diff
@@ -0,0 +1,40 @@
+# va-content
+Component update children styles
```

Comment:
We may want to add that this component applies vuestic specific typography and table styles.

Reply:
Sorry, you can explain in more detail?

Reply:
Documentation on this one will be handled later.